api/routes/monitoring_routes.py

```
@@ -0,0 +1,135 @@
+    """
+    returns the logs of the daily job
+
+    Returns:
+        string: serialized json
+    """
+    log_level = request.headers.get("level")
+    return json.dumps(read_logs("../logs/calculate_daily_activity.log", log_level))
+
+
+@monitoring_server.route("/logs/jobs/monthly")
+def get_controller_logs():
```

Ruff

Redefinition of unused `get_controller_logs` from line 115

(F811)